internal/weldr/api.go

```diff
@@ -2284,30 +2284,27 @@ func (api *API) blueprintsTagHandler(writer http.ResponseWriter, request *http.R
 // repositories for the depsolving. The actual distro object name may not correspond to the alias.
 // Since the solver uses the distro name to namespace cache, it is important to use the same distro
 // name as the one used to get the repositories.
-func (api *API) depsolve(packageSets map[string][]rpmmd.PackageSet, distroName string, arch distro.Arch) (map[string][]rpmmd.PackageSpec, map[string][]rpmmd.RepoConfig, error) {
+func (api *API) depsolve(packageSets map[string][]rpmmd.PackageSet, distroName string, arch distro.Arch) (map[string]dnfjson.DepsolveResult, error) {
 
 	distro := arch.Distro()
 	platformID := distro.ModulePlatformID()
 	releasever := distro.Releasever()
 	solver := api.solver.NewWithConfig(platformID, releasever, arch.Name(), distroName)
 
-	depsolvedSets := make(map[string][]rpmmd.PackageSpec, len(packageSets))
-	repoConfigs := make(map[string][]rpmmd.RepoConfig)
+	depsolvedSets := make(map[string]dnfjson.DepsolveResult, len(packageSets))
 
 	for name, pkgSet := range packageSets {
-		// TODO: SBOM support could be added here
```

Comment:
Nitpick: the comment is IMHO still relevant, although we probably won't ever add it to the Weldr API :innocent: